doc/cli.markdown

```diff
@@ -309,7 +306,7 @@ Examples:
 
 #### -v, --verbose
 
-add extra columns in the tabular output (SLUG)
+No-op since release v12.0.0
```

Comment:
It looks like `npm run build` was run with `BALENA_CLI_VERSION_OVERRIDE=12.0.0`. The plan is to do this for the `change-type: major` PR, but this PR is just `patch`.

Reply:
Oh damn, sorry!